pkg/scalers/selenium_grid_scaler.go

```diff
@@ -30,50 +29,83 @@ type seleniumGridScalerMetadata struct {
 	triggerIndex int
 
 	URL                 string `keda:"name=url,                      order=triggerMetadata;authParams"`
+	Username            string `keda:"name=username,                 order=triggerMetadata;authParams, optional"`
+	Password            string `keda:"name=password,                 order=triggerMetadata;authParams, optional"`
```

Comment:
actually here I am not sure we should add username and password to the trigger metadata, we tend to put these into TriggerAuthentication

Reply:
I agreed and updated it accordingly, only keeping it as authParams
I also added authParams `authType` and `accessToken`, it would be helpful in case people manage the Grid with Ingress in front and using other auth type like Bearer or OAuth2, they can provide type and access token easily.

Reply:
@zroubalik, can you please review one more round to see if all are fine? Then I can finalize and take update for keda-docs accordingly. Thanks!